node-require/add.js

```diff
@@ -0,0 +1,8 @@
+exports.add = function sumValues(a, b) {
+
+  if (isNaN(parseInt(a)) || isNaN(parseInt(b))) {
+    return 'invalid input';
+  }
+  return parseInt(a) + parseInt(b);
+
+};
```

Comment:
The calculator program should also produce the correct output for numbers with decimal places.